benchmark/asts.js

```diff
@@ -23,7 +23,7 @@ function slug(name) {
 }
 
 module.exports = FILES.map(function (file) {
-    var astJson = fs.readFileSync(FILES_PATH + '/' + slug(file) + '-ast.json');
+    const astJson = fs.readFileSync(`${FILES_PATH  }/${  slug(file)  }-ast.json`);
```

Comment:
```suggestion
    const astJson = fs.readFileSync(`${FILES_PATH}/${slug(file)}-ast.json`);
```

Reply:
I've fixed in my rebased branch, but in any case, avoiding now from this PR, so marking as resolved.